### DISCUSSION TITLE
bug: No definitive way to ask uBO Feature Requests

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#2315] [Ideas]
### Description

 **There's no other way to ask feature requests other than [r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/) support forum**. Not all uBlockOrigin users have (or) intend to use Reddit, because:
* Reddit is a closed source social media platform (unlike Discourse, Flarum, Vanilla, etc)
* [Reddit's privacy policy](https://www.reddit.com/policies/privacy-policy#policy-h2-1) is very unintuitive & hideous. It collects the user data, IP address, user-agent string, browser type, operating system, referral URLs, device information (e.g., device IDs), device settings, mobile carrier name, pages visited, links clicked, the requested URL, and search terms. And mines them for the moderation of their feed & use it for their sponsored promotions.
* Reddit was never been a non-profit company (similar to FAANG). This ofcourse, matters to many uBlockOrigin users too.

Usually feature requests need to be accepted wherever the source code is hosted, which is GitHub issue section in this case. GitHub itself is a closed source code host. But seriously, uBO itself is hosted on it, so I can't comment on [it's privacy policy](https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement) regarding user's agreement.

### Steps to Reproduce

* [No indication of feature request template under gorhill/uBlock
](https://user-images.githubusercontent.com/83843264/194268319-16142b51-0023-48e9-8bde-2cee585e5e7d.png)
* [No indication of feature request template under uBlockOrigin/uBlock-issues
](https://user-images.githubusercontent.com/83843264/194270795-69975112-1468-4cf9-ac3e-ef657521d74a.png)
* Both have questionaire & support templates that link to r/uBlockOrigin, which I don't own a reddit account & never intend to register for privacy/closed/security reasons.

### Expected behavior

Either 
* [gorhill/uBlock](https://github.com/gorhill/uBlock/issues/new/choose) or [uBlockOrigin/uBlock-issues](https://github.com/uBlockOrigin/uAssets/issues/new/choose) should **have a feature request issue template**
* Or **host an open source community forum** like discourse, etc. 
* Or just give us a Feature Request category under [Discussions](https://github.com/uBlockOrigin/uBlock-issues/discussions). 

### Actual behavior

Instead, uBlockOrigin accepts feature requests relies only on Reddit, which is a closed source, profit based social media company.

**Comment by mapx-** [comment]
I'll convert this "issue" in a discussion (where you could ask / discuss a feature)

**Reply by ghost** [reply]
Thank you for your consideration. This should actually be a Discussion rather than an Issue. Since privacy & open sourcability's importance is collectively a social topic.

**Reply by MasterKia** [reply]
"_Social topic_" => Then the better term to use is "Free software".
Because "Open Source" only cares about practical advantages while Free software cares about user freedom and privacy.

https://gnu.org/philosophy/free-sw.html
https://gnu.org/philosophy/open-source-misses-the-point.en.html

**Reply by ghost** [reply]
You're right. But the actual term is Free & Open Source Software (FOSS), according to [Open Source Definition](https://opensource.org/docs/osd), as well as Richard Stallman.
There are free softwares, that are proprietary too (I'm not talking about freeware). Privacy is achieved only when it's both free & open source.

**Reply by MasterKia** [reply]
> But the actual term is Free & Open Source Software (FOSS), according to Richard Stallman.

Not at all.

_Instead of FOSS, we say, free software or free (libre) software._
https://gnu.org/philosophy/words-to-avoid.html#FOSS
https://gnu.org/philosophy/floss-and-foss.html

**Reply by ghost** [reply]
Welp, [Neutraility](https://www.gnu.org/philosophy/words-to-avoid.html#FLOSS) is my goal after all. I donno why I should'nt be using the term FOSS or FLOSS, as GNU didn't gave us a reason why it doesn't stand for freedom. Privacy isn't about Freedom, because you intend to be private, that implies you yourself are not open & want to hide yourself, which is not Freedom to others, but rather a secrecy holder. The real freedom lies when both free & openability are combined.
If I'd be truly be supporting full freedom, it would be Liberal. On the flip side, if it's completely proprietary, it would be considered conservative. And I don't want to fight for either side, I'd rather just be neutral after all, which is what FOSS is.
Also, the term "free software" has serious issues. Caz both FOSS & FS have term "free" in it. According to GNU's philosophy, they actually have to call it "Freedom Software", instead of confusing people. Because free/libre ≠ Freedom.

**Reply by krystian3w** [reply]
So these all invalid: https://github.com/uBlockOrigin/uBlock-issues/labels/enhancement (aka https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue+is%3Aopen+label%3Aenhancement)
